### ISSUE TITLE
[Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#31442]
#### VFS team name
Authenticated Experience

#### VFS product name
Notification Preferences

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
The button "Yes, remove my information" has a different accessible name than its visible label when attempting to delete contact information. For voice assistive tech users, a discrepency between accessible and visible labels can cause confusion and lack of access by voice command. For example, someone might say "Click yes, remove my information button"-- but if the accessible name is "Remove fax number", the command may not register.

### Link, screenshot or steps to recreate
1. Log in to staging using test user 36 and turn on your screen reader
2. On personal and contact information, click "remove" for any contact information (e.g. fax number)
3. Tab to the button "Yes, remove my information" and confirm the accessible name that is announced (e.g. Remove fax number) is different than the visual label.

### VA.gov Experience Standard
[Category Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4_A

---

### Platform Recommendation
The `aria-label` matches the visible label. It can either be "Yes, remove my information" or "Remove [contact type]" as long as the `aria-label` matches the visible label.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by adamwhitlock1** [comment]
This aria-label has been updated and can be validated now

**Comment by MarciMcGuireGCIO** [comment]
@joshkimux hey!  If you have a few minutes, do you mind taking a peek at this issue to see if the 508 issue has been resolved?  Thank you!

**Comment by joshkimux** [comment]
<img width="829" alt="Screen Shot 2022-01-21 at 8 59 22 AM" src="https://user-images.githubusercontent.com/14154792/150540108-fb2192a3-ab9d-4d03-b9fd-0d14f0256e2a.png">

Great work @adamwhitlock1 ! 🥳

**Comment by MarciMcGuireGCIO** [comment]
🥳 Thanks again @adamwhitlock1 and @joshkimux ! You rock!!
